src/app/about/about.component.html

```diff
@@ -0,0 +1,53 @@
+<hp-page-banner
+  [pageTitle]="pageData.pageTitle"
+  [bannerImgUrl]="pageData.bannerUrl"
+/>
+<div class="hp-about-container">
+  <img [src]="pageData.photo" alt="doctor_img" class="hp-about-doctor-img" />
+  <div class="hp-content-container">
+    <p class="hp-welcome-text">Welcome to our Hospital!</p>
+    <h1 class="hp-title">Best Care for Your Good Health</h1>
+    <div class="hp-priorities-container">
+      <div
+        *ngFor="let priority of pageData.priorities"
+        class="hp-priority-container"
+      >
+        <div class="hp-circle"></div>
+        <p>{{ priority }}</p>
+      </div>
+    </div>
+    <p>{{ pageData.description }}</p>
+  </div>
+</div>
+
+<div class="hp-top-doctors-content-container">
+  <h3>Best doctors of the week</h3>
+  <div class="hp-top-doctors-container">
+    <app-doctor-card
+      *ngFor="let topDoctor of topDoctors | async"
+      [doctor]="topDoctor"
+    />
+  </div>
+  <hp-styled-button
+    btnText="See all doctors"
+    btnType="button"
+    (btnClick)="onNavigate()"
+  />
+</div>
+
+<div class="hp-specialties-content-container">
+  <p class="hp-subtitle">Always Caring</p>
+  <h2>Our Specialties</h2>
+  <div class="hp-specialties-container">
+    <div
+      *ngFor="let specialty of specialties | async"
+      class="hp-specialty-container"
+    >
+      <img
+        src="https://raw.githubusercontent.com/NikaKlokava/hospital-portal-assets/main/svgs/heart.svg"
+        alt="heart_svg"
```

Comment:
If you decided to provide alt, provide it as a comprehensive description of an image

Reply:
fixed it